index.html

```diff
@@ -26,15 +26,17 @@ <h1 class="name">Hannah Zeng</h1>
           </div>
         </div>
 
-        <div id="nav-bar">
-          <ul>
-            <li><a href="#about">About</a></li>
-            <li><a href="#experience">Experience</a></li>
-            <li><a href="#skills">Skills</a></li>
-            <li><a href="#projects">Projects</a></li>
-            <li><a href="#connect">Connect</a></li>
-          </ul>
-        </div>
+        <nav>
```

Comment:
Great job! If you want to improve further and follow the best practices of semantic HTML, you can use the `nav` tag instead of the `div`, like this:

```html
<nav id="nav-bar">
  <ul>
    ...
  </ul>
</nav>
```